server.go

```diff
@@ -259,7 +259,7 @@ func (s *Server) recv4(c *ipv4.PacketConn) {
 				ifIndex = cm.IfIndex
 			}
 			if err := s.parsePacket(buf[:n], ifIndex, from); err != nil {
-				// log.Printf("[ERR] zeroconf: failed to handle query v4: %v", err)
+				log.Printf("[ERR] zeroconf: failed to handle query v4: %v", err)
```

Comment:
Please revert. As long as there is no good solution to enable/disable logging on demand, zeroconf (being a lib) should be quiet.

Reply:
I agree it should be quiet but errors are exceptional and should not be ignored.

Reply:
In normal operation this will never happen, so most library users will never see any messages. But if the system is misconfigured and sockets are not working, then the user will know why instead of wondering why nothing is working.

Reply:
You're completely right. There should be correct error handling. But regarding this PR, I would like to keep those logs masked.

Reply:
Okay I will rebase to remove this commit.